### PR TITLE
Fixed Server Pong

### PR DESCRIPTION
**Pull request by PhilipShilling**
Jython, If you could add me back to the org that'd be great :)

As of Minecraft 0.11.x, there is a new way of pinging the server. This includes (but is not limited to) the version of MCPE, and the number of people online. This is a basic implementation of that.

### CODE REVIEW

**Comment by jython234** [comment]
This has been already added on the network branch. And yes, I will add you
back.

On Wednesday, June 10, 2015, Philip Shilling <notifications@github.com>
wrote:

> Jython, If you could add me back to the org that'd be great :)
>
> As of Minecraft 0.11.x, there is a new way of pinging the server. This
> includes (but is not limited to) the version of MCPE, and the number of
> people online. This is a basic implementation of that.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/BlockServerProject/BlockServer/pull/145
> Commit Summary
>
>    - Fixed Server Pong
>
> File Changes
>
>    - *M*
>    src/main/java/org/blockserver/net/protocol/pe/login/RaknetUnconnectedPong.java
>    <https://github.com/BlockServerProject/BlockServer/pull/145/files#diff-0>
>    (6)
>
> Patch Links:
>
>    - https://github.com/BlockServerProject/BlockServer/pull/145.patch
>    - https://github.com/BlockServerProject/BlockServer/pull/145.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BlockServerProject/BlockServer/pull/145>.
>


--

File: src/main/java/org/blockserver/net/protocol/pe/login/RaknetUnconnectedPong.java [hunk 1]
```diff
@@ -7,7 +7,11 @@
 public class RaknetUnconnectedPong implements PeProtocolConst{
 	private ByteBuffer bb;
 	public RaknetUnconnectedPong(long ping, long server, byte[] magic, String name){
-		name = "MCCPP;Demo;" + name;
+		int protocol_version = 27; //Minecraft PE Protocol Version
```
Comment (PEMapModder) [inline src/main/java/org/blockserver/net/protocol/pe/login/RaknetUnconnectedPong.java]: Why not camel humps?

Comment (PhilipShilling) [inline src/main/java/org/blockserver/net/protocol/pe/login/RaknetUnconnectedPong.java]: Huh?

Comment (PEMapModder) [inline src/main/java/org/blockserver/net/protocol/pe/login/RaknetUnconnectedPong.java]: `protocolVersion`

Comment (PhilipShilling) [inline src/main/java/org/blockserver/net/protocol/pe/login/RaknetUnconnectedPong.java]: Oh. I just threw that together in a minute or two. No reason as to why (or why not) I did it the way I did.

**Pull request closed without merging**